modules/extrafunctions.js

```diff
@@ -1 +1,38 @@
-﻿
\ No newline at end of file
+import {
+  title, author, addBooks, booklist, deleteBook,
+} from './add_remove.js';
+
+import { booksArr } from './onload.js';
+
+// eslint-disable-next-line no-unused-vars
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+document.querySelector('.Books-Container').addEventListener('click', (e) => {
+  booksArr.splice(e.target.id, 1);
+  localStorage.setItem('bookArr', JSON.stringify(booksArr));
+  deleteBook(e.target);
+});
+
+const addBtn = document.querySelector('#add-Button');
+const Title = document.querySelector('.title');
+
+addBtn.addEventListener('click', (e) => {
+  if (Title.value === '' || author.value === '') {
+    e.preventDefault();
+  } else {
+    const theBook = {
+      title: title.value,
+      author: author.value,
+    };
+    booksArr.push(theBook);
+    addBooks(booksArr);
+    localStorage.setItem('booksArr', JSON.stringify(booksArr));
+    document.querySelector('.emptymsg').style.display = 'none';
+    booklist.style.border = '1px solid black';
+  }
+});
```

Comment:
- [ ] New book object created override the previously added book object. At this moment, you can only add one book object to the app. Kindly look into what the issue is and fix.

[app-behavior.webm](https://user-images.githubusercontent.com/26597432/206988832-1305158c-e7a3-4996-8133-ddc91ff8fc0c.webm)

Reply:
New book adds on to previous books.